### PR TITLE
Added installation instructions to doc pages.

### LINKED ISSUE #73: Installation instructions missing in RTD
Installation instructions can be found in the README file only. They should appear in RTD too. 

To fix this issue:

- [ ] Add installation instructions to index in RTD.

### PR DESCRIPTION
**Pull request by ojeda-e**
Added installation instruction to documentation pages to fix #73.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -52,6 +54,43 @@ Why MembraneCurvature?
 in Python 3 to derive 2D maps of membrane curvature from MD Simulations, using the most recent version of `MDAnalysis`_ 
 Are you interested in calculating mean and Gaussian curvature from MD simulations? This tool is for you!
 
+
+Installation
+--------------
+
+The main dependency in MembraneCurvature is `MDAnalysis`_. You can find
+instructions to install the latest stable version of MDAnalysis via `conda`_ in the `User Guide`_.
```
Comment (lilyminium) [inline docs/index.rst:62]: Could you link directly to the installation notes in the user guide?

**Review by lilyminium — APPROVED** [review]
Generally looks good to me!

File: docs/index.rst [hunk 2]
```diff
@@ -52,6 +54,43 @@ Why MembraneCurvature?
...
+
+MembraneCurvature is available via pip:
+
+.. code-block:: bash
+
+   pip install membrane-curvature
+
+To install from source:
+
+.. code-block:: sh
```
Comment (orbeckst) [inline docs/index.rst:72]: why not also bash?

**Review by orbeckst — APPROVED** [review]
lgtm

File: docs/index.rst (hunk 2 above)
Comment (ojeda-e) [reply docs/index.rst:72]: I think I added this very last-minute change to check how does it look vs bash and forgot to revert changes. I'll leave it as bash, thanks!

File: docs/index.rst (hunk 1 above)
Comment (ojeda-e) [reply docs/index.rst:62]: Sure, I'll fix it! :)

**Pull request merged**